File: C Language/Sum_Of_Natural_Number.c
//Sum of Natural Number using Recursion

#include<stdio.h>

int sum(int x)
{
    if(x != 0)
    {
        return x + sum(x-1);
    }
    else
    {
        return x;
    }
}

int main()
{
    int num, result;
    printf("Enter a number:\n");
    scanf("%d",&num);
    
    result = sum(num);
    printf("\nSum is %d", result);
    return 0;
}

File: Python/Sum_Of_Natural_Number.py
class SumOfNN:
	def _init__(self):
		pass
		
	def sum(self, x):
		if x != 0:
			return x + self.sum(x-1)
		else:
			return x
			
sonn = SumOfNN()

print('Enter a number:')
num = int(input())

result = sonn.sum(num)
print()
print('Sum is {}'.format(result))